packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -9,6 +9,7 @@
                 :value="address.line1"
                 :class="$style['c-address-formField']"
                 name="address-line-1"
+                maxlength="255"
```

Comment:
might be worth moving it to a reusable variable?

Reply:
Unsure the benefit can you expand on this?